.bazelversion

```diff
@@ -0,0 +1 @@
+5.0.0
```

Comment:
Why is this particular version of Bazel pinned?

Reply:
@Maratyszcza  It was the version of Bazel I used to test this change. In the meantime, I tested all versions of Bazel up to 6.3.2 successfully.

We could express this using skylib:

```
load("@bazel_skylib//lib:versions.bzl", "versions")

versions.check("5.0.0", "6.3.2")
```
see e.g. [here].(https://github.com/Vertexwahn/BazelDemos/blob/main/core_principles/bazel_version/WORKSPACE.bazel)

or just get rid of the `.bazelversion` file.

My main intention was just to document which version of Bazel was used to test this - we could also add a comment to the `.bazelversion` file:

```
5.0.0
# This was the version which was used to test those changes. Should also work with newer versions of Bazel.
```

This works, since Bazelisk only reads the first line (there is no real support for comments in `.bazelversion` files)

Could be that this change is broken with Bzlmod which is maybe introduced in Bazel 7.0.0.

In the end there is no real need for exact Bazel 5.0.0 and probably this works also with Bazel 2.0.0 and earlier...

Reply:
@Maratyszcza - so there are 4 options:
- keep it as it is
- add a comment to `.bazelversion` file why this specific version and what other version work...
- remove the .bazelversion file
- use skylib for version check (but this needs an update every time a new Bazel version is released)